src/main/java/se/sundsvall/byggrintegrator/api/ByggrIntegratorResource.java

```diff
@@ -60,13 +44,20 @@ public ByggrIntegratorResource(ByggrIntegratorService byggrIntegratorService) {
 	}
 
 	@GetMapping(path = "/byggr/neighborhood-notification/{identifier}/errands", produces = { APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE })
```

Comment:
Of course a matter of personal opinion, but the `/byggr` part of the path could be moved to the class-level `@RequestMapping` annotation